hybrasyl/Objects/Creature.cs

```diff
@@ -386,12 +386,25 @@ public virtual bool Turn(Direction direction)
 
             foreach (var obj in Map.EntityTree.GetObjects(GetViewport()))
             {
-                if (!(obj is User)) continue;
-                var user = obj as User;
-                var x11 = new ServerPacket(0x11);
-                x11.WriteUInt32(Id);
-                x11.WriteByte((byte)direction);
-                user.Enqueue(x11);
+                if (obj is User)
+                {
+                    var user = obj as User;
+                    var x11 = new ServerPacket(0x11);
+                    x11.WriteUInt32(Id);
+                    x11.WriteByte((byte)direction);
+                    user.Enqueue(x11);
+                }
+                if (obj is Monster)
+                {
+                    var mob = obj as Monster;
+                    var x11 = new ServerPacket(0x11);
+                    x11.WriteUInt32(Id);
+                    x11.WriteByte((byte)direction);
+                    foreach (var user in Map.EntityTree.GetObjects(Map.GetViewport(mob.X, mob.Y)).OfType<User>().ToList())
+                    {
+                        user.Enqueue(x11);
+                    }
+                }
```

Comment:
Didn't we say anytime we ran into these things or added more we were going to make them use `ServerPacketStructures`? *ducks, runs away*

Reply:
Yes. However this was direct copied from the user movement method and modified for monsters.

Reply:
![technically correct](http://i0.kym-cdn.com/photos/images/facebook/000/909/991/48c.jpg)